flyteplugins/go/tasks/logs/logging_utils.go

```diff
@@ -53,7 +54,7 @@ func GetLogsForContainerInPod(ctx context.Context, logPlugin tasklog.Plugin, tas
 			PodRFC3339FinishTime:      time.Unix(finishTime, 0).Format(time.RFC3339),
 			PodUnixStartTime:          startTime,
 			PodUnixFinishTime:         finishTime,
-			TaskExecutionIdentifier:   taskExecID,
+			TaskExecutionID:           taskExecID,
```

Comment:
Backwards compatibility?

Reply:
Couldn’t think of issues with backward compatibility since this will only affect new log links. Maybe I’m missing something?

Reply:
I mean, the Input object becomes your interface, right? if somebody is referencing `{.TaskIdentifier.Project}` that will no longer resolve if I'm reading correctly, no?

Reply:
As discussed offline, the external-facing template variables remain unchanged.